src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.ts

```diff
@@ -83,21 +83,10 @@ export class UserFormComponent implements OnInit {
     });
     if (this.mode === this.userFormMode.editing) {
       this.initEdit();
-    } else {
-      this.initAdd();
     }
   }
 
```

Comment:
Could you please add a test for the minimum required filled input fields?

Reply:
The only required field ATM is the `username`, and we already have a test for that field:

https://github.com/ceph/ceph/blob/bdc93a3550b9e2d4f4ad41cd818da713d94bfc43/src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.spec.ts#L88

What to you really want to test?

Reply:
Just if the form is invalid if the username is not set, and that it is valid if it is set.